File: DKImagePickerControllerDemo_OC/DKImagePickerControllerDemo_OC/CustomUIDelegate/CustomGroupDetailCameraCell.m
//
//  CustomGroupDetailCameraCell.m
//  DKImagePickerControllerDemo_OC
//
//  Created by 赵铭 on 2017/7/10.
//  Copyright © 2017年 zm. All rights reserved.
//

#import "CustomGroupDetailCameraCell.h"

@implementation CustomGroupDetailCameraCell
+ (NSString *)cellReuseIdentifier{
    return @"CustomGroupDetailCameraCell";
}

- (instancetype)initWithFrame:(CGRect)frame{
    if (self = [super initWithFrame: frame]) {
        UILabel * cameraLabel = [[UILabel alloc] initWithFrame:frame];
        cameraLabel.text = @"Camera";
        cameraLabel.textAlignment = NSTextAlignmentCenter;
        cameraLabel.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
        [self.contentView addSubview:cameraLabel];
    }
    return self;
}
/*
// Only override drawRect: if you perform custom drawing.
// An empty implementation adversely affects performance during animation.
- (void)drawRect:(CGRect)rect {
    // Drawing code
}
*/

@end

File: Example/DKImagePickerControllerDemo/CustomUIDelegate/CustomGroupDetailCameraCell.swift
//
//  CustomGroupDetailCameraCell.swift
//  DKImagePickerControllerDemo
//
//  Created by ZhangAo on 03/01/2017.
//  Copyright © 2017 ZhangAo. All rights reserved.
//

import UIKit
import DKImagePickerController

class CustomGroupDetailCameraCell: DKAssetGroupDetailBaseCell {
    
    class override func cellReuseIdentifier() -> String {
        return "CustomGroupDetailCameraCell"
    }
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        
        let cameraLabel = UILabel(frame: frame)
        cameraLabel.text = "Camera"
        cameraLabel.textAlignment = .center
        cameraLabel.autoresizingMask = [.flexibleWidth, .flexibleHeight]
        self.contentView.addSubview(cameraLabel)
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
}

